app/controllers/interviews_controller.rb

```diff
@@ -53,6 +60,13 @@ def destroy
 
   private
 
+  def set_headers
+    headers['Access-Control-Allow-Origin'] = '*'
+    headers['Access-Control-Expose-Headers'] = 'ETag'
+    headers['Access-Control-Allow-Methods'] = 'GET, POST, PATCH, PUT, DELETE, OPTIONS, HEAD'
+    headers['Access-Control-Allow-Headers'] = '*,x-requested-with,Content-Type,If-Modified-Since,If-None-Match'
+  end
+
```

Comment:
Do we need this? CORS can be directly configured in application.rb. Please check this. Discuss if needed.